### PR TITLE
TINY-5940: Updated dependencies

### PR DESCRIPTION
**Pull request by tiny-james**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tinymce/tinymce-react",
-  "version": "3.5.0",
+  "version": "3.5.1",
```
Comment (lnewson) [inline package.json:3]: Any reason you're bumping the version here as we're only updating the dev dependencies? Also if you are releasing a new version, then we also need to update the changelog.

- [ ] Changelog or leave this as is for now

Comment (tiny-james) [reply package.json:3]: Ok, I thought we had to do a release. I will remove the version bumps.

Comment (lnewson) [reply package.json:3]: Yeah, I should have clarified that earlier sorry!

Comment (tinydylan) [reply package.json:3]: Can you please release a new version?

Comment (lnewson) [reply package.json]: @tinydylan what's the purpose? Just to pull in the fix done the other day for the `utils` maybe?

Comment (tinydylan) [reply package.json:3]: As discussed out-of-band, since this resolves a security alert, we should release an update.

Comment (lnewson) [reply package.json:3]: I wasn't included in the discussion and it was my concern, so please include me next time :unamused:

**Pull request merged**